lib/uri_parser.js

```diff
@@ -225,6 +227,7 @@ function applyConnectionStringOption(obj, key, value, options) {
   } else if (key === 'appname') {
     value = decodeURIComponent(value);
   } else if (key === 'readconcernlevel') {
+    obj['readConcernLevel'] = value;
```

Comment:
what is happening here?

Reply:
this does look like it could just be added to `CASE_TRANSLATION` above?

Reply:
What we currently do with `readConcernLevel` is set `readConcern: { level: value }`. The spec tests check that we have `readConcernLevel: value` so that's why I am pulling the `value` up a level.

Reply:
@daprahamian can this be resolved?